### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by germartinez**
Add docs

### CODE REVIEW

**Comment by rmeissner** [comment]
Is this somewhere linked?

File: docs/cpk.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+```
+static async create(opts?: CPKConfig): Promise<CPK>
+```
+Static method that creates and initializes an instance of the CPK with the selected configuration parameters.
+
+## constructor
+```
+constructor(opts?: CPKConfig)
+```
+Creates a non-initialized instance of the CPK with the selected configuration parameters.
```
Comment (cag) [inline docs/cpk.md:13]: Could maybe document `async init(): Promise<void>` as well.

File: docs/cpk.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+## isProxyDeployed
+```
+async isProxyDeployed(): Promise<boolean>
+```
+Returns true or false if the Safe Proxy used by the CPK is already deployed (at least one transaction was made) or if it is not (no transactions made yet).
```
Comment (cag) [inline docs/cpk.md:19]: What about a direct connection, either through WalletConnect or via the Safe Apps SDK, to a Safe with no transactions?

File: docs/cpk.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+```
+get ethLibAdapter(): EthLibAdapter | undefined
+```
+Returns the EthLibAdapter in use.
+
+## networks
+```
+get networks(): NetworksConfig
+```
+Returns the network configuration in use.
```
Comment (cag) [inline docs/cpk.md:43]: "network**s** configuration"? Maybe call it a listing of the addresses of contracts which drive the CPK per network by network ID?

**Review by cag — APPROVED** [review]
It's a good bit of starting content.

**Pull request merged**